recipes/libdrs_c/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "libdrs" %}
+{% set version = "2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-uvcdat-{{ version }}.tar.gz
+  url: https://github.com/UV-CDAT/libdrs/archive/master.tar.gz
+  sha256: dc057cedbcc0bfcd3aafdc3e04aca9c428f08d6fa1d679778d64be75980462a5
+
+build:
+  number: 0
+  skip: True  # [win] 
+
+requirements:
+  build:
+    - libnetcdf 4.4.1
```

Comment:
Please check the proper conda-forge pinning: https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies